### PR TITLE
platfrom: byoc standard architecture update to show one vpc

### PR DESCRIPTION
**Pull request by wojcik-dorota**
https://aiven.atlassian.net/browse/DOC-712

### CODE REVIEW

File: docs/platform/concepts/byoc.rst
Comment (akudiyar) [inline docs/platform/concepts/byoc.rst:65]: > Aiven accesses this VPC via a static IP address

via -> from?

> Aiven utilizes a bastion host logically separated

As it is a VM in s different subnet, can we talk here about it being _physically_ separated?

> You are able to run your services

Maybe "you are able to integrate your services". The old phrasing here seems to be more accurate 

Should we write here also that the service nodes connect to the Aiven services and so they need an internet access via NAT, while the bastion service utilizes an internet gateway?

Comment (wojcik-dorota) [reply docs/platform/concepts/byoc.rst]: @akudiyar , thank you :-) Implemented all bit for the last one. Since we decided to make it a simplified view and removed the outbound from the diagram (both NAT GW and Internet GW not visible), I'd say we better leave them out from the description for now. We can cover that as soon as we have move detailed tech architecture in place. Hope that makes sense?

**Review by bugant — COMMENTED** [review]
This looks good to me.  I'd anyway wait for reviews from @heikju and @akudiyar.

**Review by bugant — APPROVED** [review]

**Review by heikju — APPROVED** [review]

**Review by ArthurFlageul — APPROVED** [review]

**Pull request merged**
